src/test/java/net/openhft/chronicle/queue/impl/single/stress/RollCycleMultiThreadStressSharedWriterQueueTest.java

```diff
@@ -32,6 +32,6 @@ public void stress() throws Exception {
     }
 
     public static void main(String[] args) throws Exception {
-        new RollCycleMultiThreadStressSharedWriterQueueTest().stress();
+        new RollCycleMultiThreadStressSharedWriterQueueTest().main();
```

Comment:
`main` would be a static method wouldn't it? Can we just do:

```
RollCycleMultiThreadStressSharedWriterQueueTest.main(args);
```

Otherwise it looks like we're creating an instance unnecessarily

Reply:
Oh never mind, that's very confusing though... perhaps the old naming was less confusing?

Reply:
Or even `run()`?

Reply:
Yep OK fair enough I will change to run